src/gui/newwizard/setupwizardwindow.cpp

```diff
@@ -149,6 +151,18 @@ void SetupWizardWindow::slotStartTransition()
     slotHideErrorMessageWidget();
 }
 
+void SetupWizardWindow::reject()
+{
+    auto messageBox = new QMessageBox(QMessageBox::Warning, tr("Cancel Setup"), tr("Do you really want to cancel the account setup?"), QMessageBox::Yes | QMessageBox::No, ocApp()->gui()->settingsDialog());
```

Comment:
This message doesn't make a lot of sense to me terminology wise. We consider this the "setup wizard".

Reply:
Yes but the wizard does set up an account?